src/pages/exam.tsx

```diff
@@ -141,13 +162,25 @@ const Exam = () => {
               </li>
             ))}
           </div>
-          <button type="submit">Siguiente</button>
+          <div>
+            {currentQuestionIndex > 0 && (
+              <button onClick={handlePreviousQuestion} type="submit">
+                Anterior
```

Comment:
Please put everything in the UI in English

Reply:
Done